### PR TITLE
MAUI-1951: How to migrate from Xamarin Forms to .NET MAUI.(SfLinearGauge, SfRadialGauge and Barcode)

### PR DESCRIPTION
**Pull request by SVRagul**
Hi @sheiksyedm,

I have completed the review changes in the migration ocument of the SfLinearGauge, SfRadialGauge and Barcode. Can you please review and merge this.

Regards,
Ragul S V

### CODE REVIEW

File: MAUI/Barcode-Generator/migration.md [hunk 1]
```diff
@@ -9,9 +9,9 @@ documentation: ug
 
 # Migrate from Xamarin.Forms SfBarcode to .NET MAUI SfBarcodeGenerator
 
-To migrate easier from [Xamarin SfBarcode](https://www.syncfusion.com/xamarin-ui-controls/xamarin-barcode) to [.NET MAUI SfBarcodeGenerator](https://www.syncfusion.com/maui-controls/maui-barcodes), we kept most of the APIs from Xamarin SfBarcode in MAUI SfBarcodeGenerator. However, to maintain the consistency of API naming in MAUI SfBarcodeGenerator, we renamed some of the APIs. The APIs that have been changed in MAUI SfBarcodeGenerator from Xamarin SfBarcode are detailed as follows.
+To make the migration from the [Xamarin SfBarcode](https://www.syncfusion.com/xamarin-ui-controls/xamarin-barcode) to [.NET MAUI SfBarcodeGenerator](https://www.syncfusion.com/maui-controls/maui-barcodes) easier, we kept most of the APIs from the Xamarin SfCircularGauge in the.NET MAUI SfRadialGauge. However, to maintain the consistency of API naming in the.NET MAUI SfRadialGauge, we renamed some of the APIs. Please find the difference in the below topics.
```
Comment (sheiksyedm) [inline MAUI/Barcode-Generator/migration.md:12]: @SVRagul  Use barcode controls name here.

File: MAUI/Linear-Gauge/migration.md [hunk 2]
```diff
@@ -9,9 +9,9 @@ documentation: ug
 
 # Migrate from Xamarin.Forms SfLinearGauge to .NET MAUI SfLinearGauge
 
-To migrate easier from [Xamarin SfLinearGauge](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfGauge.XForms.SfLinearGauge.html) to [.NET MAUI SfLinearGauge](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Gauges.SfLinearGauge.html), we kept most of the APIs from Xamarin SfLinearGauge in .NET MAUI SfLinearGauge. However, to maintain the consistency of API naming in .NET MAUI SfLinearGauge, we renamed some of the APIs. The APIs that have been changed in MAUI SfLinearGauge from Xamarin SfLinearGauge are detailed as follows.
+To make the migration from the [Xamarin SfLinearGauge](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfGauge.XForms.SfLinearGauge.html) to [.NET MAUI SfLinearGauge](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Gauges.SfLinearGauge.html) easier, we kept most of the APIs from the Xamarin SfCircularGauge in the.NET MAUI SfRadialGauge. However, to maintain the consistency of API naming in the.NET MAUI SfRadialGauge, we renamed some of the APIs. Please find the difference in the below topics.
```
Comment (sheiksyedm) [inline MAUI/Linear-Gauge/migration.md:12]: Use linear gauge controls name here.

File: MAUI/Barcode-Generator/migration.md (hunk 1 above)
Comment (SVRagul) [reply MAUI/Barcode-Generator/migration.md]: Used the respective control name.

File: MAUI/Linear-Gauge/migration.md (hunk 2 above)
Comment (SVRagul) [reply MAUI/Linear-Gauge/migration.md]: Used the respective control name.

**Review by sheiksyedm — APPROVED** [review]

**Pull request merged**